tests/Exception/HttpClientExceptionTest.php

```diff
@@ -34,4 +34,30 @@ public function testBadRequestGetMessage()
         $exception = HttpClientException::badRequest($response);
         $this->assertStringEndsWith('<html><body>Server HTML</body></html>', $exception->getMessage());
     }
+
+    public function testForbiddenRequestThrowException()
+    {
+        $response = new Response(403, ['Content-Type' => 'application/json'], '{"Error":"Business Verification"}');
+        $exception = HttpClientException::forbidden($response);
+        $this->assertInstanceOf(HttpClientException::class, $exception);
+        $this->assertSame(403, $exception->getCode());
+    }
+
+    public function testForbiddenRequestGetMessageJson()
+    {
+        $response = new Response(403, ['Content-Type' => 'application/json'], '{"Error":"Business Verification"}');
+        $exception = HttpClientException::forbidden($response);
+        $this->assertStringEndsWith('Business Verification', $exception->getMessage());
+
+        $response = new Response(403, ['Content-Type' => 'application/json'], '{"Message":"Business Verification"}');
+        $exception = HttpClientException::forbidden($response);
+        $this->assertStringEndsWith('{"Message":"Business Verification"}', $exception->getMessage());
+    }
+
+    public function testForbiddenRequestGetMessage()
+    {
+        $response = new Response(400, ['Content-Type' => 'text/html'], '<html><body>Forbidden</body></html>');
+        $exception = HttpClientException::badRequest($response);
```

Comment:
Hey @Littlesqx

Are you sure that we should consider a `bad request` exception when we are testing a `forbidden` implementation?

Cheers,

Reply:
@DavidGarciaCat Oh, no~~~sorry for my carelessness. I didn't change the code after paste `bad request test case` 😿